__TESTS__/unit/actions/Overlay.test.ts

```diff
@@ -14,12 +14,12 @@ describe('Tests for overlay actions', () => {
       })
       .resize(fill(1000, 1000))
       .overlay(
-        overlayLayer(
-          imageSource('sample')
+        imageLayer(
+          image('sample')
```

Comment:
Shouldn't we make it possible to import only source.image instead of whole source ?

Reply:
Yes, we should, I'm not entirely sure what's the bundle impact on keeping it together.

For now, Source seems to be a dumb wrapper around TransforambleImage.

Reply:
Lets not have a "for now" for overlay's source at this stage, we should finalize it's spec

Reply:
We will have different types of sources - image,text,video,subtitles etc no? lets allow to import only whats needed

Reply:
This is related to the above comment, Obviously that's the goal, but we're missing the right place to put Source right now.

Source is not an action, so I'm wondering where we should place it to make it meaningful export.